processlib/activity.py

```diff
@@ -114,6 +125,18 @@ def finish(self, **kwargs):
         self.instance.save()
         self._instantiate_next_activities()
 
+    def finish_process(self):
```

Comment:
Can we make this `_finish_process` - we don't want others calling this directly. To finish a process you finish the activities, and they take care of the rest.